governance-aepp/src/utils/aeternity.js

```diff
@@ -147,7 +154,7 @@ aeternity.initWalletSearch = async (successCallback) => {
     name: 'AEPP',
     nodes: [
       {name: 'ae_mainnet', instance: await Node({url: settings.ae_mainnet.nodeUrl})},
-      //{name: 'ae_uat', instance: await Node({url: settings.ae_uat.nodeUrl})}
+      {name: 'ae_uat', instance: await Node({url: settings.ae_uat.nodeUrl})}
```

Comment:
should we leave this active?

Reply:
good question, let me check

Reply:
it should be included, just removed it earlier for compatibility, but seems to be a non-issue now